Add App to Teams and Tenant/Add App to Teams or Tenant  using Graph API.ps1

```diff
@@ -0,0 +1,96 @@
+param(
+      [Parameter(Mandatory=$true)][System.String]$OwnerPrincipalName,
+      [Parameter(Mandatory=$true)][System.String]$AppName,
+      [Parameter(Mandatory=$true)][System.String]$Tenantid,
+      [Parameter(Mandatory=$true)][System.String]$client_Id,
+      [Parameter(Mandatory=$true)][System.String]$Client_Secret
+      )
+      
+Connect-MicrosoftTeams
```

Comment:
Script has no logging, might be good to add debug log capabilities as its a multi-step process. https://github.com/TeamsAdminSamples/PowerShell/blob/main/z_Templates/New-PowerShell.ps1 as a read-to-use logging function that can easily integrated.

Reply:
Added login to the script.